src/watcher.rs

```diff
@@ -14,30 +14,63 @@ const DEBOUNCE_DELAY: Duration = Duration::from_millis(100);
 const FALLBACK_POLLING_TIMEOUT: Duration = Duration::from_secs(1);
 
 pub fn watch(path: PathBuf, changed: SyncSender<()>) {
+    let Ok(path_metadata) = path.metadata() else {
+        error!("config file {path:?} is not valid");
```

Comment:
I use `error!` in niri for programming errors that are nevertheless recoverable. So these all should be `warn!` since the config path is controlled by the user.